monkey/monkey_island/cc/services/telemetry/processing/__init__.py

```diff
@@ -1,7 +0,0 @@
-# import all implemented hooks, for brevity of hooks.py file
-from .tunnel import process_tunnel_telemetry
-from .state import process_state_telemetry
-from .exploit import process_exploit_telemetry
-from .scan import process_scan_telemetry
-from .system_info import process_system_info_telemetry
-from .post_breach import process_post_breach_telemetry
```

Comment:
we don't need this file at all then

Reply:
We always need an init file to make it a package